src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheck.java

```diff
@@ -37,7 +38,7 @@
     implements FileSetCheck {
 
     /** Collects the error messages. */
-    private final LocalizedMessages messageCollector = new LocalizedMessages();
+    private final SortedSet<LocalizedMessage> errorMessages = new TreeSet<>();
```

Comment:
Renaming will be done in #4465 when we introduce CheckstyleViolation as a part of our model.